tests/test_parser.py

```diff
@@ -45,8 +45,8 @@ def test_dictionary_attr(data: dict[str, Attribute]):
 
 
 @irdl_attr_definition
-class TestAttr(ParametrizedAttribute):
-    name = 'test.attr'
+class DummyAttr(ParametrizedAttribute):
```

Comment:
What is the issue with `TestAttr`? Is this used somewhere else?

Reply:
Just pytest does not like names starting with Test:

```
tests/test_parser.py:47
  /home/gb4018/workspace/xdslproject/xdsl/tests/test_parser.py:47: PytestCollectionWarning: cannot collect test class 'TestAttr' because it has a __init__ constructor (from: tests/test_parser.py)
    @irdl_attr_definition
```